frappe_private_comment/public/js/footer.bundle.js

```diff
@@ -0,0 +1,133 @@
+// NOTE: run `bench build` after making changes to this file
+// alternatively, run `bench watch` to automatically build on file changes
```

Comment:
Need this? We can do this with DOM only, same as I added the visibility icons 🤔 ?

Reply:
footer.bundle.js build is required to override the refresh by frappe